core/terraform/json-to-env.sh

```diff
@@ -101,6 +101,10 @@ jq -r '
         {
             "path": "event_grid_airlock_notification_topic_resource_id",
             "env_var": "EVENT_GRID_AIRLOCK_NOTIFICATION_TOPIC_RESOURCE_ID"
+        },
+        {
+            "path": "arm_environment",
```

Comment:
Why do we need this?

Reply:
The DefaultAzureCredential is used when no managed identity exists- so basically on local debugging in our use case, for this we need the TF output param to be added to private.env so the api will have it when debugging.  Reference: core/terraform/outputs.sh

Reply:
But I think this file and the outputs.sh one are used to deal with settings coming for terraform. Why do we need to "export" this one from terraform if it already exists as a regular environment sourced directly from the config?

Reply:
Following our conversation, defined arm env as required param in the config. So it will be loaded to the env